internal/api/v1/admin/transactions/admin_transaction_filter_query_builder.go

```diff
@@ -0,0 +1,28 @@
+package transactions
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/querybuilders"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/transactions"
+	"github.com/bitcoin-sv/spv-wallet/models/filter"
+)
+
+type adminTransactionFilterQueryBuilder struct {
+	transactionFilter filter.AdminTransactionFilter
+}
+
+func (a *adminTransactionFilterQueryBuilder) Build() (url.Values, error) {
+	builder := transactions.TransactionFilterBuilder{
+		TransactionFilter:  a.transactionFilter.TransactionFilter,
+		ModelFilterBuilder: querybuilders.ModelFilterBuilder{ModelFilter: a.transactionFilter.ModelFilter},
+	}
+	params, err := builder.BuildExtendedURLValues()
+	if err != nil {
+		return nil, fmt.Errorf("failed to build extended URL values: %w", err)
+	}
+
+	params.AddPair("xpubid", a.transactionFilter.XPubID) // xpubid should be replaced by xpubId in filter model.
```

Comment:
What this question in comment refers to?
It it's a TODO - add a JIRA task for that.

Reply:
Basically, it's not a TODO comment rather a note describing discrepancy in naming. Most of admin filter models has 'xpubId' tag name instead of 'xpubid'.  Generally, minor thing :)